iphone/Classes/TiUIListView.m

```diff
@@ -1815,7 +1815,24 @@ - (void)scrollViewWillBeginDragging:(UIScrollView *)scrollView
         [self.proxy fireEvent:@"dragstart" withObject:nil withSource:self.proxy propagate:NO reportSuccess:NO errorCode:0 message:nil];
     }
 }
-
+-(void)scrollViewWillEndDragging:(UIScrollView *)scrollView withVelocity:(CGPoint)velocity targetContentOffset:(inout CGPoint *)targetContentOffset{
```

Comment:
LOL.
Will do my best to get this ready before Monday. Greetings from Amsterdam, Hans!